mempool.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"bytes"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+func HandleTransaction(verb, hash string) {
+	if verb != "POST" && verb != "DELETE" {
```

Comment:
why `"DELETE"`? you don't use it in the code

Reply:
The API has a method for [adding](https://docs.blocknative.com/mempool-tools/webhook-api#add-transaction-to-watch) and [removing](https://docs.blocknative.com/mempool-tools/webhook-api#remove-transaction-to-watch) transaction to watch. I grouped them in one function with a method parameter.

Reply:
ok, but then you don't use `verb` in the code